### PR TITLE
Add initial config of /docs to use gatsby theme

### LINKED ISSUE #9: Initial configuration of /docs to use the Gatsby theme
**Is your feature request related to a problem? Please describe.**
The /docs directory is an example of a site that uses our Gatsby theme locally

**Describe the solution you'd like**
Setup a folder with the Gatsby dependencies and the theme created in #6 
Something like [this](https://www.gatsbyjs.com/docs/themes/building-themes/)

### PR DESCRIPTION
**Pull request by JGuilhermeCoelho**
#### What is the purpose of this pull request?
The /docs directory was created with gatsby and theme dependencies, the theme previously named /theme, has now been renamed to /andromedocs-theme

#### What problem is this solving?
Resolves #9 

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[merge]
```
Comment (thayannevls) [inline .gitconfig:1]: This file is personal, should be located at `~/` on your computer

File: andromedocs-theme/public/page-data/index/page-data.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"componentChunkName":"component---src-pages-index-mdx","path":"/","result":{"pageContext":{"frontmatter":{"title":"Hello world!","path":"/page","date":"2020-10-01T00:00:00.000Z","author":"Vtex"}}},"staticQueryHashes":[]}
```
Comment (thayannevls) [inline andromedocs-theme/public/page-data/index/page-data.json:1]:
```suggestion
{"componentChunkName":"component---src-pages-index-mdx","path":"/","result":{"pageContext":{"frontmatter":{"title":"Hello world!","path":"/page","date":"2020-10-01T00:00:00.000Z","author":"VTEX"}}},"staticQueryHashes":[]}
```

could you also format the json please? :pleading_face:

**Review by JRobsonJr — COMMENTED** [review]
I'm thinking we could add `public/` to our .gitignore, right? I don't think we need that folder stored in Git, since it's part of the build :thinking: Does it make sense?

**Comment by JGuilhermeCoelho** [comment]
> I'm thinking we could add `public/` to our .gitignore, right? I don't think we need that folder stored in Git, since it's part of the build  Does it make sense?

I've been thinking about it for a while, I think so

File: .gitconfig (hunk 1 above)
Comment (JGuilhermeCoelho) [reply .gitconfig]: resolved 4b90437

File: andromedocs-theme/public/page-data/index/page-data.json (hunk 2 above)
Comment (JGuilhermeCoelho) [reply andromedocs-theme/public/page-data/index/page-data.json]: resolved 4b90437

**Review by thayannevls — APPROVED** [review]
Awesome :rocket:

**Review by JRobsonJr — APPROVED** [review]
Well done!! :rocket:

**Review by LeandraOliveiraS — APPROVED** [review]
Great job :rocket:

**Pull request merged**